csvcubed/tests/unit/cli/inspect/test_metadatainputvalidator.py

```diff
@@ -0,0 +1,109 @@
+from csvcubed.cli.inspect.metadatainputvalidator import CSVWType, MetadataValidator
+from csvcubed.cli.inspect.metadataprocessor import MetadataProcessor
+import pytest
```

Comment:
Generally with imports we're not *too* strict, but try to keep the external dependencies in a separate block to the internal project dependencies.

e.g.

```
import pytest

from csvcubed.cli.inspect.metadatainputvalidator import CSVWType, MetadataValidator
from csvcubed.cli.inspect.metadataprocessor import MetadataProcessor
from tests.unit.test_baseunit import get_test_cases_dir
```

> Imports should be grouped in the following order:
>
>    Standard library imports.
>    Related third party imports.
>    Local application/library specific imports.
>
> You should put a blank line between each group of imports.

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
@robons I noticed that I dont need the `pytest` import in that test file. So I've removed it. But I have updated all the other files as per the python import guidelines you have referenced.